### ISSUE TITLE
PHP version problem

### ISSUE THREAD

**Issue by designjpp** [teamneusta/pimcore-backend-branding-bundle#1]
Hello, 

I tried to test the bundle, but it is not compatible with PHP 8.3

```
Package teamneusta/pimcore-backend-branding-bundle has requirements incompatible with your PHP version, PHP extensi
  ons and Composer version:
    - teamneusta/pimcore-backend-branding-bundle 0.9.0 requires php ~8.1.0 || ~8.2.0 which does not match your instal
  led version 8.3.10.
```

**Comment by jdreesen** [comment]
In fact, I have never tested the bundle with Pimcore 11 :see_no_evil:
I will do this in the next few days and then also add support for PHP 8.3.

**Comment by designjpp** [comment]
Greats. Thank you!

**Comment by jdreesen** [comment]
It seems like it works as it should with Pimcore 11 :)

I just released a new version with PHP 8.3 support: https://github.com/teamneusta/pimcore-backend-branding-bundle/releases/tag/0.9.1
